.gitignore

```diff
@@ -1,25 +1,32 @@
-# Ignore everything
-/*
+# Gradle
+.gradle
 
-# Subprojects
-!/fabric-*
-/fabric-*/.gradle
-/fabric-*/build
+# Eclipse
+.classpath
+.metadata
+.settings
+.project
+*.launch
 
-# Folders
-!/gradle
-!/src
+# Intellij/Idea
+.factorypath
+.idea
+*.iml
+*.ipr
+*.iws
 
-# Files
-!/.gitattributes
-!/.gitignore
-!/.editorconfig
-!/build.gradle
-!/gradle.properties
-!/gradlew
-!/gradlew.bat
-!/HEADER
-!/Jenkinsfile
-!/LICENSE
-!/README.md
-!/settings.gradle
+# Build artifacts
+bin/
+build/
+jars/
+
```

Comment:
Tested locally on current Fabric project and needed to add `out/`

Otherwise perfect - eclipse wise.